lib/pengine/pe_output.c

```diff
@@ -20,6 +20,7 @@ pe__name_and_nvpairs_xml(pcmk__output_t *out, bool is_list, const char *tag_name
     CRM_ASSERT(tag_name != NULL);
 
     xml_node = pcmk__xml_peek_parent(out);
+    CRM_ASSERT(xml_node != NULL);
```

Comment:
pcmk__xml_peek_parent allows to peek an emtpy stack, so the user takes the responsibility to check its not NULL.